apps/console/src/features/users/components/wizard/invite-parent-org-user-wizard.tsx

```diff
@@ -78,17 +162,22 @@ export const InviteParentOrgUserWizard: FunctionComponent<InviteParentOrgUserWiz
             </Modal.Header>
             <Modal.Content className="content-container" scrolling>
                 { alert && alertComponent }
-                <InviteParentOrgUser
-                    closeWizard={ closeWizard }
-                    setIsSubmitting={ setIsSubmitting }
-                    onUserInviteSuccess={ onUserInviteSuccess }
-                    setAlert={ setAlert }
-                    data-componentid={ `${ componentId }-form` }
-                />
+                { parentOrgUserInvitationResults && parentOrgUserInvitationResults?.length > 0
```

Comment:
We don't need && if we are using ?. operator.

The purpose of ?. (optional chaining) operator is to get rid of && chains. :) 

```suggestion
                { parentOrgUserInvitationResults?.length > 0
```

Reply:
Removed

Reply:
Removed